### PR TITLE
Updates to orbit ui [READY]

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
Pretty big orbit ui refactor.
Check changelog for full list of features.

<details>
<summary>screenshots</summary>

In game
Screenshot 2024-05-15 133935

Sort by department
Screenshot 2024-05-15 133951

Did you know ninjas had a hud icon that hasn't worked for four years?
image

Criticals
image

Orbit blade

https://github.com/tgstation/tgstation/assets/42397676/99681548-bfb3-4895-9c95-3b650df71107

</details>

## Why It's Good For The Game
Some QoL for the orbit menu, giving more info on where action is and isn't. 
Removes more of the uselocalstate hook which is deprecated anyways
## Changelog
:cl:
fix: Fixed an issue preventing space ninjas from having a hud icon
add: ORBIT UI CHANGES:
add: AFK players are greyed out.
add: Living NPCs now display health.
add: Icons displayed are now based on hud icons, which includes icons for player-visible antagonists
add: You can now sort by job department (click health icon)
add: Round ending "critical" items will be listed at the top.
add: Click the settings button to expand for more info
add: Your current orbit target is highlighted.
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
are afk players still selectable?

**Comment by jlsnow301** [comment]
> are afk players still selectable?

entirely, their buttons are gray (just like you'd see in misc)

**Comment by norsvenska** [comment]
just me, but i much prefer the fontawesome icons over the sechud icons. the sechud ones just don't fit with the rest of the ui imo, while the fa ones are designed to mesh well.

**Comment by PapaMichae1** [comment]
Mixed feelings, I don't love that it's bigger, and I think the ability to color people by department kinda completely eliminates the need to have big colored icons

**Comment by jlsnow301** [comment]
> Mixed feelings, I don't love that it's bigger, and I think the ability to color people by department kinda completely eliminates the need to have big colored icons

bigger how?

**Comment by PapaMichae1** [comment]
I felt like it took up more vertical space than before but I checked again and I was wrong, whoops

**Comment by jlsnow301** [comment]
antags are getting icons

File: code/modules/antagonists/space_ninja/space_ninja.dm [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	name = "\improper Space Ninja"
 	antagpanel_category = ANTAG_GROUP_NINJAS
 	job_rank = ROLE_NINJA
-	antag_hud_name = "space_ninja"
+	antag_hud_name = "ninja"
```
Comment (jlsnow301) [inline code/modules/antagonists/space_ninja/space_ninja.dm:5]: incredible

**Comment by jlsnow301** [comment]
Currently a bug where clicking the bottom half of the button will click the one below it (???)

**Pull request merged**